todo-app/src/todo.js

```diff
@@ -4,21 +4,19 @@ import TaskDate from './TaskDate.js'
 import State from './state.js'
```

Comment:
The filename should be the same as the component class name, so in this case `Todo.js`. This is just to be consistent with the rest of the project. Other projects might follow other conventions.